src/csharp/Grpc.Core.Tests/ClientServerTest.cs

```diff
@@ -139,6 +139,26 @@ public void UnaryCall_ServerHandlerSetsStatus()
             Assert.AreEqual(0, ex2.Trailers.Count);
         }
 
+        [Test]
+        public void UnaryCall_StatusDebugErrorStringNotTransmittedFromServer()
+        {
+            helper.UnaryHandler = new UnaryServerMethod<string, string>((request, context) =>
+            {
+                context.Status = new Status(StatusCode.Unauthenticated, "", "this DebugErrorString value should not be transmitted to the client");
+                return Task.FromResult("");
+            });
+
+            var ex = Assert.Throws<RpcException>(() => Calls.BlockingUnaryCall(helper.CreateUnaryCall(), "abc"));
+            Assert.AreEqual(StatusCode.Unauthenticated, ex.Status.StatusCode);
+            Assert.IsTrue(ex.Status.DebugErrorString.Contains("Error received from peer"));  // a different debug error string set by grpc client
```

Comment:
nit: searching for the string "Error received from peer" seems like a fine sanity check for the test, but since it isn't really stable, and might change e.g. if the error message from C-core changes, maybe add an actionable error message to this assert, something like:
```
Assert.IsTrue(ex.Status.DebugErrorString.Contains("Error received from peer"), "Is \"Error received from peer\" still a valid substring to search for in the client-generated error message from C-core?");
```

Reply:
Done.